### ISSUE TITLE
Discussion Post on GitHub Issues: Where Might We Go from Here?

### ISSUE THREAD

**Issue by ebeshero** [ebeshero/DHClass-Hub#200]
Choose one of the following Greensburg project sites under development. Describe (in your own words) at least one significant topic it seems to be exploring, and reflect on what we might want to try next to develop the research areas of the project and/or the UX of the project site. Those of us (instructors and returning students) who have been involved in these projects should help introduce them to our class and point out things that need work.
* [Decameron Project](http://decameron.newtfire.org)</ref>: Developed by coding students at Pitt-Greensburg last spring, and ready to be continued.

* [Amadis in Translation](http://amadis.newtfire.org): Recently launched project last year: follow the link to its GitHub repository for more information and a view of the coding. We need to work on preparing the two sets of English and Spanish base texts to read and study them together, and we could choose to expand the project in a new direction of interest.
                    
* From our Digital Mitford Project: [Letters
                        Introduction](http://digitalmitford.org/letters.html): For the letters portion of this project, a team could develop a module to help investigate some interesting topics. 
* [Digital Archives and Pacific Cultures](http://pacific.pitt.edu/) : This site reflects the origins of our Pitt-Greensburg digital humanities projects—it was our starting point and first collaboration with students. The Voyage Maps area got us involved with using maps to represent passages from voyage logs, but the project could use an overhaul to pick up loose ends or continue where we left off a few years ago, and we could move it to our development space at the Supercomputing Center to be with our other projects.

**Comment by setriplette** [comment]
Amadis in Translation is my project, and it's just a skeleton right now. We are attempting to compare the early modern version to Southey's translation, and even with just six chapters under examination, we are starting to see that what motivates changes between source and translation is often cultural factors. Southey is trying to explain medieval Spain to his English audience and making all sorts of "errors" that are quite interesting. The project needs everything, but the most urgent question is to get more chapters into xml. We started with print book images, not a digital text. Madee from our class is helping us with this!

**Comment by etj27** [comment]
I looked at the DeCameron project, it appears to be trying to explore the aspects of perspective in finding out who is the narrator for each section of the work.  This is somewhat similar to Piez's work on Frankenstein, which is also another way of interpreting the story.  As far as developing the site, I suppose the UX experience could be improved in some areas (perhaps color scheme in the top menu) but overall probably doing something with the text to make it easier to read.  Other than that its looking good.

**Comment by jonhoranic** [comment]
I explored the Digital Archives and Pacific Cultures site (which in my opinion could use a re-branding, but that is just a small nitpick) that contains various historical documents centered around different travels and interactions with native Pacific islander cultures. There are a lot of separate things the site is trying to accomplish.
1. Archive and create markup that translate into highlights for different documentation.
2. Place coordinate markers on a map with descriptions to visualize voyages.
3. Create charts and graphs to condense mark up information in an easy to measure section.
I can see that some pages on the site have yet to fully integrated GUI's on every page, some documents have not had been coded into the archive, there are more markups to translate into graphs, and maps still need plotted out. There can be some small design and polish tweaks in between to help make it more appealing and accessible to readers. This site has the potential to be a wide database of rare and obscure parts of our worlds history to come to the surface in an easy to navigate web platform. The more people that can learn and be engaged with the content within the better.

**Comment by ahunker** [comment]
I had the pleasure of looking around on the Decameron Project and from the start I can tell how much work and time have gone into creating it and I applaud those involved for all their effort. The way the website is laid out, the objective is super clear as it's right there on the homepage and the rest of the site reflected it. While looking through the different options on the drop-down menu, one significant topic that was clearly present, was how the Decameron is structured and presented. The explanation alone of how the group chose to present the model is a great example but seeing it marked within the texts themselves really drove it home and helped me understand the text a little bit better, even while skim reading. 
 One thing that I struggled a bit with, though, was the color scheme and the text itself. The dark green text looked a bit faded and failed to grab attention along the drop-down menu and the bright colors used to mark the structure, while attention grabbing, made it difficult for me to read what was highlighted. I also found the actual text to be small and bunched together, and I found myself squinting or struggling to read through the large block paragraphs. 
 Aside from a few aesthetic features, I think the website has great potential and I look forward to seeing how the project develops.
